plugins/funind/g_indfun.mlg

```diff
@@ -148,7 +148,7 @@ END
 module Vernac = Pvernac.Vernac_
 module Tactic = Pltac
 
-type function_rec_definition_loc_argtype = (Vernacexpr.fixpoint_expr * Vernacexpr.decl_notation list) Loc.located
+type function_rec_definition_loc_argtype = Vernacexpr.fixpoint_expr Loc.located
```

Comment:
Not sure there's much need for this alias now that it's simpler, maybe just unfold it at the 1 place it's used.

Reply:
Sounds good, done!